README.md

```diff
@@ -24,6 +24,20 @@ gulp.task('sass', function () {
 });
 ```
 
+You can also compile synchronously, doing something like this:
+
+```javascript
+var gulp = require('gulp');
+var gutil = require('gulp-util');
```

Comment:
Oh, so the `it` in your comment is `gulp-util` and you're saying that `gulp-util` is still a variable in the examples even though we've moved logging to `sass.logError` and `gulp-util` should be removed as we are using `sass.logError` in our examples and is the recommended way of doing that? I understand. I'll update that.